libraries/script-engine/src/ScriptEngine.cpp

```diff
@@ -294,12 +309,13 @@ void ScriptEngine::runDebuggable() {
         if (_lastUpdate < now) {
             float deltaTime = (float)(now - _lastUpdate) / (float)USECS_PER_SECOND;
             if (!_isFinished) {
+                processNextImmediate();
```

Comment:
You will probably get comments on this w.r.t. the performance implications. Is it a good thing that this is being included in the update time?